server/src/main/java/org/elasticsearch/index/mapper/DateFieldMapper.java

```diff
@@ -168,8 +168,8 @@ NumericType numericType() {
             return numericType;
         }
 
-        ToScriptField<SortedNumericDocValues> getDefaultToScriptField() {
-            return toScriptField;
+        ToScriptFieldFactory<SortedNumericDocValues> getDefaultToScriptField() {
```

Comment:
What do you think about changing this to `getDefaultToScriptFieldFactory`?

Reply:
Yes! Done.